R/tfb-wavelet-utils.R

```diff
@@ -27,13 +27,13 @@ interpolate_arg <- function(arg_list) {
 
 
 
-fit_wavelet <- function(data, Z, least_squares, glmnet_args) {
+fit_wavelet <- function(data, Z, penalized, glmnet_args) {
```

Comment:
@SvenLorenz renaming this to "penalized" so that it's consistent with the args in tfb_spline